QuantitativeReporting.py

```diff
@@ -385,11 +385,14 @@ def setupSegmentationObservers(self):
       segNode.AddObserver(vtkSegmentationCore.vtkSegmentation.SegmentAdded, self.onSegmentAdded))
 
   def initializeWatchBox(self, node):
-    try:
-      dicomFileName = node.GetStorageNode().GetFileName()
-      self.watchBox.sourceFile = dicomFileName if os.path.exists(dicomFileName) else None
-    except AttributeError:
+    if not node:
+      self.watchBox.sourceFile = None
+      return
+    dicomFileName = slicer.dicomDatabase.fileForInstance(node.GetAttribute("DICOM.instanceUIDs").split(" ")[0])
```

Comment:
@che85 it is not impossible that this attribute is also missing. If that is the case, it means the volume was not loaded from DICOM, and there should be error communicated to the user.